scripts/docker/debian/stretch/Dockerfile.doc

```diff
@@ -31,12 +30,17 @@ RUN mkdir -p ${GTEST_ROOT} \
     && tar -zxvf gtest.tar.gz --strip-components=1 -C ${GTEST_ROOT} \
     && rm gtest.tar.gz
 
+# ronn
+# Unfortunately the package [`ruby-ronn`](https://packages.debian.org/buster/all/ruby-ronn/filelist) does not include the `ronn` executable.
```

Comment:
They have split the package, it is now called ronn https://packages.debian.org/buster/ronn